pkg/virt-controller/services/renderresources.go

```diff
@@ -537,29 +537,15 @@ func initContainerMinimalRequests() k8sv1.ResourceList {
 }
 
 func hotplugContainerResourceRequirementsForVMI(vmi *v1.VirtualMachineInstance) k8sv1.ResourceRequirements {
-	if vmi.IsCPUDedicated() || vmi.WantsToHaveQOSGuaranteed() {
-		return k8sv1.ResourceRequirements{
-			Limits:   hotplugContainerMinimalLimits(),
-			Requests: hotplugContainerMinimalLimits(),
-		}
-	} else {
-		return k8sv1.ResourceRequirements{
-			Limits:   hotplugContainerMinimalLimits(),
-			Requests: hotplugContainerMinimalRequests(),
-		}
+	return k8sv1.ResourceRequirements{
+		Limits:   hotplugContainerMinimalResources(),
+		Requests: hotplugContainerMinimalResources(),
 	}
 }
 
-func hotplugContainerMinimalLimits() k8sv1.ResourceList {
+func hotplugContainerMinimalResources() k8sv1.ResourceList {
```

Comment:
I prefer maintaining the existing pattern of `*MinimalRequests` and `*MinimalLimits` functions.  In this case the request function can call the limit function.  I just feel that is a pattern we will want to go back to eventually with a proper fix

Reply:
The name seemed wrong for what it was doing, that is why I changed it. I have no issues keeping the original naming it just didn't seem to fit with this particular change, that is why I changed the name.